src/main/webapp/scheduled-interviews.jsp

```diff
@@ -0,0 +1,30 @@
+<%@page import="java.util.ArrayList"%>
+<%@page import="java.util.List"%>
+<%@ page import="com.google.sps.data.ScheduledInterviewRequest" %>
+<%
+  List<ScheduledInterviewRequest> scheduledInterviews = (List<ScheduledInterviewRequest>)request.getAttribute("scheduledInterviews");
+  pageContext.setAttribute("scheduledInterviews", scheduledInterviews);
+%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<c:choose>
+  <c:when test= "${empty scheduledInterviews}">
+    <h2 style="text-align: center">No Scheduled Interviews</h2>
+  </c:when>
+  <c:otherwise>
+    <c:forEach items= "${pageScope.scheduledInterviews}" var="scheduledInterview">
+      <div class="row">
+        <div class="card w-75 scheduled-interview-card">
+          <div class="card-body">
+            <h5 class="card-title">Your Role: ${scheduledInterview.getRole()}</h5>
+            <p class="card-text">${scheduledInterview.getDateString()}</p>
+          </div>
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item">Interviewee: ${scheduledInterview.getIntervieweeId()}</li>
```

Comment:
This should not be an id, but rather a first name. You might need a separate class just for generating the jsp. (Replacing the id fields with names)

Reply:
Done, I changed the name of the methods in the scheduledInterviewRequest class. I use that class to get the names associated with the interviewer and interviewee ids and display them on the page.